specification/azsadmin/resource-manager/containerregistry/Microsoft.ContainerRegistry.Admin/preview/2019-11-01/configuration.json

```diff
@@ -0,0 +1,264 @@
+          "ContainerRegistryConfiguration"
+        ],
+        "description": "Returns the specified configuration details.",
+        "operationId": "ContainerRegistryConfiguration_Get",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "#/parameters/ConfigurationNameParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The configuration has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryConfiguration"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.ContainerRegistry.Admin/locations/{location}/configuration": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of configuration at the given location.": {
+            "$ref": "./examples/Configuration/ListConfiguration.json"
+          }
+        },
+        "tags": [
+          "ContainerRegistryConfiguration"
+        ],
+        "description": "Returns a list of configuration at the given location.",
+        "operationId": "ContainerRegistryConfiguration_List",
+        "parameters": [
+          {
+            "$ref": "containerregistry.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "containerregistry.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of configuration has been returned.",
+            "schema": {
+              "$ref": "#/definitions/ContainerRegistryConfigurationList"
+            }
+          },
+          "default": {
+            "description": "Error Response.",
+            "schema": {
+              "$ref": "containerregistry.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ContainerRegistryConfiguration": {
+      "description": "Container registry configuration property.",
+      "type": "object",
+      "properties": {
+        "properties": {
+          "description": "Container registry configuration property.",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ContainerRegistryConfigurationProperty"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "containerregistry.json#/definitions/Resource"
```

Comment:
use the common resource type

Reply:
We don't have access to the common types from our repo:
https://github.com/msazurestackworkloads/azure-rest-api-specs

Reply:
See response elsewhere.  It is an anti-pattern to define your own resource